SupersonicSound/Studio/System.cs

```diff
@@ -68,9 +68,11 @@ private void Dispose(bool disposing)
             {
                 if (disposing)
                 {
-                }
+                    LowLevelSystem.Dispose();
+                    LowLevelSystem = null;
 
-                _system.release().Check();
+                    _system.release().Check();
```

Comment:
To be honest I wasn't sure if these should be inside or outside the if(disposing), it just seemed to make more sense inside.